File: Dutch Open Hackathon/2015/app/KvKickstart/KvKickstart/Models/Presentations.cs
using System;
using System.Collections.Generic;
using System.Text;
using DOH2015.Framework;

namespace DOH2015.Models
{

	public class PresentationResponse
	{
		public bool success { get; set; }
		public string message { get; set; }
		public Body body { get; set; }
	}

	public class Body
	{
		public List<Presentation> presentations { get; set; }
	}

	public class Presentation
	{
		public int id { get; set; }
		public string title { get; set; }
		public string city { get; set; }
		public string time { get; set; }
		public string presenterId { get; set; }
		//public Presenter presenter { get { return KamerVanKoophandel.Presenter (int.Parse (presenterId)).Result; }}
		public Presenter presenter {get { return new Presenter() { name = "Lorem Ipsum" };}}
		public string building { get; set; }
		public string distance { get; set; }
		public string content { get; set; }
		public string imageUrl { get; set; }
	}
}

File: Dutch Open Hackathon/2015/web/app/models/Presentations.php
<?php

namespace Hackathon\Models;

class Presentations extends \Phalcon\Mvc\Model
{

    /**
     *
     * @var integer
     */
    public $id;

    /**
     *
     * @var string
     */
    public $title;

    /**
     *
     * @var string
     */
    public $city;

    /**
     *
     * @var string
     */
    public $time;

    /**
     *
     * @var integer
     */
    public $presenterId;

    /**
     *
     * @var string
     */
    public $building;

    /**
     *
     * @var string
     */
    public $distance;

    /**
     *
     * @var string
     */
    public $content;

    /**
     *
     * @var string
     */
    public $imageUrl;

    /**
     * Initialize method for model.
     */
    public function initialize()
    {
        $this->belongsTo('presenterId', 'Presentors', 'id', array('alias' => 'Presentors'));
    }

    /**
     * Returns table name mapped in the model.
     *
     * @return string
     */
    public function getSource()
    {
        return 'presentations';
    }

    /**
     * Allows to query a set of records that match the specified conditions
     *
     * @param mixed $parameters
     * @return Presentations[]
     */
    public static function find($parameters = null)
    {
        return parent::find($parameters);
    }

    /**
     * Allows to query the first record that match the specified conditions
     *
     * @param mixed $parameters
     * @return Presentations
     */
    public static function findFirst($parameters = null)
    {
        return parent::findFirst($parameters);
    }

}
